docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+- set up basic alert rules
+
+We set it up for you, out-of-the-box!
+
+
+## Monitoring with Sematext Agent
+
+Sematext Agent collects various metrics and other info about servers, virtual machines, container hosts and containers and ships that to Sematext Cloud.
+
+First create an [Infra Monitoring App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create). You will be presented with installation instructions for many possible environments.
+
+
+## How it works
+
+A Golang-based [Sematext Agent](../agents/sematext-agent) will be installed on your machine. This Agent sends all infrastructure data to an Infra App whose token is configured during the Agent's installation.
+
+It is capable of running in two kinds of environments:
+- bare-metal, virtual machine - we provide rpm/deb packages for most popular linux distributions
+- containers - a special sematext/agent container image provides support for monitoring in container environments
+
+In both cases, installing the agent will start gathering all infrastructure data right away. Additionally, it will enable [Monitoring](../monitoring/autodiscovery) and [Logs Autodiscovery](../logs/discovery/intro/) features which make it possible to automatically start monitoring any of supported [Sematext integrations](./index) or to ship logs produced by any process or container.
+
+### eBPF Support
+
+To gain deep **insight into the Linux kernel**, Sematext Agent relies on **eBPF** to implant **instrumentation points**, which means to **attach eBPF programs to kprobes** on kernel functions. This ensures a very efficient and powerful system exploration approach with better network tracing and negligible overhead.
+
+
+## Enabling Infra Monitoring
+
+If you've set up Sematext Monitoring at some point in the past, you can easily enable Infra Monitoring by:
+
+- [Upgrading](./spm-faq/#agent-updating) your `sematext-agent` to the latest version
+- Setting up Infra App token on each of your machines by running the `setup-infra` command (in bare-metal/virtual machine environments)
+
+> **Note**: For container monitoring, we suggest uninstalling the existing deprecated Agent and going through the new setup steps for Docker Monitoring in Sematext Cloud.
+
+When doing a clean installation, regardless of whether you're installing in containerized environment or not, follow monitor installation instructions displayed in Sematext Cloud. Those instructions will always include everything needed to set up an Infra App, either on its own or alongside some other App Agent.
+
+
+### The `setup-infra` command
+
+This command is applicable only in bare-metal/virtual machine environments.
+
+Setting up or changing which Infra App should receive infrastructure data and metrics from a particular machine can be done with the <b>setup-infra</b> command.
+
+To see which Infra Apps exist in your account visit Sematext Cloud -> Integrations -> Apps. You can choose any of the existing Infra Apps or create a new one.
```

Comment:
To see which Infra Apps exist in your account visit Sematext Cloud -> Integrations -> Apps. You can choose any of the existing Infra Apps or create a new one.
=> 
To see which Infra Apps exist in your account visit Sematext Cloud -> Apps.  You can choose any of the existing Infra Apps and go to App Settings > Tokens or you can create a new Infra App.